docs/content/docs/recipes/teams.md

```diff
@@ -0,0 +1,347 @@
+            getFluidContainer(url.searchParams);
+        });
+        microsoftTeams.appInitialization.notifySuccess();
+    }
+}, [inTeams]);
+```
+
+1. Replace `TODO 6` with the following code. Note here that it's important to ensure that the content page is loaded inside Microsoft Teams and that the Fluid container is defined before passing it into the React component (defined as `FluidComponent` below).
+
+```ts
+if (inTeams === false) {
+  return (
+      <div>This application only works in the context of Microsoft Teams</div>
+  );
+}
+
+if (fluidContainer !== undefined) {
+  return (
+      <FluidComponent fluidContainer={fluidContainer} />
+  );
+}
+
+return (
+  <div>Loading FluidComponent...</div>
+);
+```
+
+### Creating React component for Fluid view and data
+
+Now that you have married the basic creation flow of Teams and Fluid, you can now create your own React component that handles the interactions between the application view and Fluid data. From this point on, the logic and flow behaves just like any other Fluid-powered application. With the basic structure established, you can create any of the [Fluid examples](https://github.com/microsoft/FluidExamples) as a Teams application by changing the `ContainerSchema` and the application view's interaction with the DDSes/data objects on the content page.
+
+## Start the Fluid server and run the application
+
+If you are running your Teams application locally with Azure Client local mode, make sure to run the following command in the Command Prompt to start the Fluid service.
+
+```dotnetcli
+npx @fluidframework/azure-local-service@latest
+```
+
+To run and start the Teams application, open another terminal and follow the [instructions](https://docs.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/create-channel-group-tab?tabs=nodejs#upload-your-application-to-teams) to run the application server.
+
+Now follow the [instructions](https://docs.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/create-channel-group-tab?tabs=nodejs#upload-your-application-to-teams) to upload the application to a Teams Tab.
+
+{{< callout warning >}}
+
+Hostnames with `ngrok`'s free tunnels are not preserved. Each run will generate a different URL. This means that anytime a new `ngrok` tunnel is created, the older container will no longer be accessible. For production scenarios, please visit [the section below](#using-azureClient-with-azure-fluid-relay)
+
+{{< /callout >}}
+
+## Next steps
+
+### Using AzureClient with Azure Fluid Relay
+
+Because this is a Teams tab application, collaboration and interaction is the main focus. Consider replacing the local mode `AzureClientProps` provided above with non-local credentials from your Azure service instance, so others can join in and interact with you in the application! Check out [how to provision your Azure Fluid Relay service](https://docs.microsoft.com/en-us/azure/azure-fluid-relay/how-tos/provision-fluid-azure-portal).
```

Comment:
```suggestion
Because this is a Teams tab application, collaboration and interaction are the main focuses. Consider replacing the local mode `AzureClientProps` provided above with non-local credentials from your Azure service instance, so others can join in and interact with you in the application! Check out [how to provision your Azure Fluid Relay service](https://docs.microsoft.com/en-us/azure/azure-fluid-relay/how-tos/provision-fluid-azure-portal).
```